commit 0364d6cac3ab4c001f383b67b843667401df2de5
Author: Jakub Kicinski <jakub.kicinski@netronome.com>
Date:   Fri Feb 24 11:43:36 2017 -0800

    vxlan: lock RCU on TX path
    
    [ Upstream commit 56de859e9967c070464a9a9f4f18d73f9447298e ]
    
    There is no guarantees that callers of the TX path will hold
    the RCU lock.  Grab it explicitly.
    
    Fixes: c6fcc4fc5f8b ("vxlan: avoid using stale vxlan socket.")
    Signed-off-by: Jakub Kicinski <jakub.kicinski@netronome.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1381_0364d6cac3ab4c001f383b67b843667401df2de5.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1381_0364d6cac3ab4c001f383b67b843667401df2de5.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
